nixos/modules/installer/cd-dvd/installation-cd-base.nix

```diff
@@ -30,6 +30,11 @@ with lib;
   # Add Memtest86+ to the CD.
   boot.loader.grub.memtest86.enable = true;
 
+  # On a fresh machine, before formatting, an installation
+  # media cannot tolerate a host config defined file system layout
```

Comment:
Kind of unclear in this context that we're overriding a full fledged normal host config. Reader may not look up `mkImageMediaOverride`.

It's unnatural in English to start a sentence with location and time. These almost always go at the end of the sentence. I had trouble parsing the subject `an installation media` here.